src/liballoc/rc.rs

```diff
@@ -438,6 +437,38 @@ impl Rc<str> {
     }
 }
 
+impl<T> Rc<[T]> {
```

Comment:
Out of curiosity, for the purposes of the compiler, does `Rc<[T]>` provide a measurable improvement over `Rc<Vec<T>>`?